Add warnings after patch_sklearn & unpatch_sklearn. Update of documentation (#574)
intel/scikit-learn-intelex: doc/sklearn.rst

    import daal4py.sklearn
    daal4py.sklearn.patch_sklearn()

> If some modules from scikit-learn were imported before patch_sklearn(),

Reviewer: ```suggestion
If some modules from scikit-learn were imported before ``patch_sklearn()``,
```

> then you need to re-import these modules after patch_sklearn().

Reviewer: ```suggestion
you have to reimport these modules after ``patch_sklearn()``.
```

Reviewer 2: since you repeat the name of the function twice in one sentence, you might want to rephrase it, e.g. something like this:

```suggestion
you have to reimport these modules again after patching.
```

or

```suggestion
you have to reimport these modules again after enabling the patches.
```

It is possible to undo the patch with::

    daal4py.sklearn.unpatch_sklearn()

> After unpatch_sklearn(), you need to re-import the scikit-learn modules.

Reviewer: ```suggestion
After applying ``unpatch_sklearn()``, reimport the scikit-learn modules.
```
